jts-transaction-service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:22
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /home
+
+ENV NARAYANA_PORT 4710
+ENV NARAYANA_VERSION 5.2.1.Final
+ENV NARAYANA_HOME /home/narayana
+ENV JACORB_LOG_LEVEL 3
```

Comment:
Still doesn't work. Now URL is:
```
https://www.jboss.org/jbosstm/downloads/%225.2.1.Final%22=NARAYANA_HOME=%22/opt/jboss/narayana%22
```
I'll try to update my docker client, that might be an issue